### DISCUSSION TITLE
Documentation about projection scale

### DISCUSSION THREAD

**Discussion by stockiNail** [sgratzl/chartjs-chart-geo#77] [Q&A]
In the README, the following sentence is wrtten about projection scale:

> A new scale projection is registered and used by default by Choropleth and BubbleMap. It provides just one option to specify the projection method. 

Nevertheless, having a look to `IProjectionScaleOptions`, there are other 2 ooptions, not documented:

https://github.com/sgratzl/chartjs-chart-geo/blob/71aee67748d35f6bc9b3247574e9b0e9e6cb80a9/src/scales/ProjectionScale.ts#L72-L73

Should we consider them as "private"? If not, do you think it's possible to document them, explaining their meaning and usage?

**Comment by sgratzl** [comment]
both options provide an additional way to scale/translate the projection. see also source at:

https://github.com/sgratzl/chartjs-chart-geo/blob/71aee67748d35f6bc9b3247574e9b0e9e6cb80a9/src/scales/ProjectionScale.ts#L160-L162

an example would be to move and scale the map on top of the automatic scaling:

https://codepen.io/sgratzl/pen/abJgroe?editors=0110

**Comment by stockiNail** [comment]
Thank you very much @sgratzl 
Do you think the readme should be changed in order to remove the sentence  `It provides just one option` because not correct?

**Comment by stockiNail** [comment]
If  I may, always in readme, `radiusScale` is mentioned but I think it is called `SizeScale` and I don't see any `radiusScale` options.

**Reply by sgratzl** [reply]
I changed it in the dev branch.

**Comment by joedual** [comment]
Hello. Is there any possibility to center the map on a certain city (latitude and longitude) regardless of the use of projectionOffset?

**Reply by sgratzl** [reply]
looking at https://github.com/d3/d3-geo#projection_center there is a center method but I haven't tried it yet
